src/components/plan/PlanModelErrorBar.svelte

```diff
@@ -0,0 +1,54 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let hasErrors: boolean = false;
+
+  let isVisible: boolean = hasErrors;
+
+  $: isVisible = hasErrors;
+
+  const dispatch = createEventDispatcher<{
+    viewModelErrors: void;
+  }>();
+
+  function onClickViewConsole() {
+    dispatch('viewModelErrors');
+  }
+
+  function onDismiss() {
+    isVisible = false;
+  }
+</script>
+
+{#if isVisible}
+  <div class="model-error-bar">
+    <div class="info"></div>
+
+    <div class="buttons">
+      <button on:click={onClickViewConsole} class="st-button secondary view-button">View errors in console</button>
+      <button class="st-button secondary" on:click={onDismiss}>Dismiss</button>
+    </div>
+  </div>
+{/if}
+
+<style>
+  .model-error-bar {
```

Comment:
I'd prefer some styling on the error bar to differentiate it from the plan snapshot bar
<img width="1306" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/2d2fb4c5-9766-488e-aaed-323d7ab9a656">
Mock:
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/d917ff4b-6592-4280-9376-e12dff159f74)

Reply:
Yeah...I totally forgot to finish the model error bar lol